editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1098,6 +1099,46 @@ void SpriteFramesEditor::_animation_add() {
 	animations->grab_focus();
 }
 
+void SpriteFramesEditor::_animation_duplicate() {
+	if (updating) {
+		return;
+	}
+
+	if (!frames->has_animation(edited_anim)) {
+		return;
+	}
+
+	int counter = 1;
+	String new_name = String(edited_anim);
```

Comment:
```suggestion
	String new_name = edited_anim;
```
Doesn't need to be explicit.